lib/common/observability-builder.ts

```diff
@@ -3,12 +3,17 @@ import * as utils from '@aws-quickstart/eks-blueprints/dist/utils';
 import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 
+type ComputeType = "ec2" | "fargate";
```

Comment:
Also please remove any changes to this file and handle it from ourside, we will soon remove this file and use the blueprints observability builder